tools/topology/topology2/development/tplg-targets.cmake

```diff
@@ -84,17 +84,71 @@ EFX_DRC_COMPONENT=multiband,EFX_MBDRC_PARAMS=passthrough"
 # With 16 kHz DMIC1
 "sof-hda-generic\;sof-hda-generic-cavs25-4ch-48k-16k\;HDA_CONFIG=mix,\
 PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-hda-generic-cavs25-4ch-48k-16k.bin,\
-NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,DMIC1_ENABLE=passthrough,DMIC1_RATE=16000"
+NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,DMIC1_ENABLE=passthrough,\
+DMIC1_RATE=16000,DMIC1_PCM_ID=24"
 
 "sof-hda-generic\;sof-hda-generic-ace1-4ch-48k-16k\;HDA_CONFIG=mix,PLATFORM=mtl,\
 PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-hda-generic-ace1-4ch-48k-16k.bin,\
-NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,DMIC1_ENABLE=passthrough,DMIC1_RATE=16000"
+NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,DMIC1_ENABLE=passthrough,\
+DMIC1_RATE=16000,DMIC1_PCM_ID=24"
```

Comment:
Why not "PCM11  capture	DMIC16kHz" ?

Reply:
I changed it because I understood from your earlier comment that 11 for WoV. This 24 would be "PCM24  capture Voice  (Communication with a human)" but I'll change this back to 11.